kube_deploy/Dev/deploy.yml

```diff
@@ -74,6 +74,11 @@ spec:
                 secretKeyRef:
                   name: cica-web-secrets
                   key: cw_maintenance_message_enabled
+            - name: CW_OS_PLACES_API_KEY
```

Comment:
Given this is visible in the request header, does this need to be a bona fide secret?

Reply:
Yeah, requests the os-api are proxied via cica-web and will be sent over https. And we don't want to expose this key in our repo.